metrics/metrics.go

```diff
@@ -21,6 +21,10 @@ import (
 // for less cluttered pprof profiles.
 var Enabled bool = false
 
+// EnabledStr has the same function as Enabled but
+// it can be set during compilation (linking) time.
+var EnabledStr = "false"
```

Comment:
Does it serve a purpose to enable metrics for build via `-ldflags '-X ..'` build flag?

If so, how does it relate to "be able to enable metrics through config"?